pkgs/applications/video/mpv/scripts/autoload.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv-autoload";
+  version = "2019-02-15";
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/mpv-player/mpv/e162bcb5a09b5ad514636684314e8e797ab2a47b/TOOLS/lua/autoload.lua";
+    sha256 = "0fs04dzz7zcprprsmhnwzn6ma6x6v3jkyx8yyf0rdcna51ndrxbk";
+  };
+  dontBuild = true;
+  dontUnpack = true;
+  installPhase = ''
+    mkdir -p $out/share/mpv/scripts
+    cp ${src} $out/share/mpv/scripts/autoload.lua
+  '';
+  passthru.scriptName = "autoload.lua";
+
+  meta = {
+    description = "This script automatically loads playlist entries before and after the the currently played file";
+    homepage = "https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua";
+    maintainers = [ lib.maintainers.dawidsowa ];
+    license = lib.licenses.gpl2Plus;
+    longDescription = ''
```

Comment:
I don't know if that's needed. Upstream already documents this inside the lua script, and `longDescription` should be a description, not a "how to use" instructions.